README.md

```diff
@@ -117,10 +117,10 @@ $ vagrant up
 # ssh into the Vagrant machine, and setup the dev environment
 $ vagrant ssh
 $ cd /kong
-$ make dev
+$ make dev -i
 
 # only if you want to run the custom plugin, tell Kong to load it
-$ export KONG_CUSTOM_PLUGINS=myplugin
+$ export KONG_PLUGINS=myplugin
```

Comment:
this is not correct, since it will not load the bundled plugins. I think we need a version note here.

Use `export KONG_PLUGINS=bundled,myplugin`, and for versions `< 0.14` use `export KONG_CUSTOM_PLUGINS=myplugin`

Reply:
Yes. That will be better for all case. Thanks.

Reply:
Additional this, I think it's necessary to add some note about the version previous this step because the first step only make the clone to the repo of kong, and that define version for default.